easybuild/easyconfigs/a/ASE/ASE-3.15.0-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.15.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5e22d961b1311ef4ba2d83527f7cc7448abac8cf9bddd1593bee548459263fe8']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('matplotlib', '2.1.0', '-Python-%(pyver)s'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ase'],
+    'dirs': [],
```

Comment:
existing ASE easyconfigs have a more extensive `sanity_check_paths`:

```python
sanity_check_paths = {
    'files': ['bin/ase-build', 'bin/ase-db', 'bin/ase-gui', 'bin/ase-info', 'bin/ase-run'], (1/1)
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
}
```

It doesn't include `bin/ase` yet though, it makes sense to include that too imho...

Reply:
Hmm, i thought i took that from latest ase .eb but perhaps not from develop...